src/test/java/config_loaders/UserConfigLoader.java

```diff
@@ -0,0 +1,34 @@
+package config_loaders;
+
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class UserConfigLoader {
+    private Properties properties;
+
+    public UserConfigLoader() {
```

Comment:
Don't repeat yourself. Constructor of this class and of EnvironmentConfigLoader differs only by properties file name. All duplicating code must be moved into separate method.